charts/nfs/.helmignore

```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```

Comment:
Instead of having a copy of this chart, could we check if we can use the official one?

E.g.

```
helm upgrade nfs-provider \
    --install stable/nfs-server-provisioner \
    --set persistence.enabled=true
```

Reply:
The nfs we are using today is built for GCP, and has the flag :
`volumes:
        - name: mypvc
          gcePersistentDisk:`
To identify the gcp disk. Not sure how this would be done with `nfs-server-provisioner `
Any idea? 

Another option worth considering is to move the backing disk to Cloud Filestore which seems to be the recommended way for nfs. Then we would be able to only deploy the helm chart `nfs-client-provisioner` 
What do you think?

Reply:
If the second option isn't too much work, that sounds nice. Is there any drawbacks to cloud filestore?